common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -84,15 +85,24 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
       soundcloud.attr("height", "300") // height is necessary if data-visual == true
     }
 
+    // There are 2 url patterns that have been encountered in soundcloud embeds.
     def getTrackIdFromUrl(soundcloudUrl: String): Option[String] = {
-      val pattern = ".*soundcloud.com%2Ftracks%2F(\\d+).*".r
+      val pattern = ".*api.soundcloud.com%2Ftracks%2F(\\d+).*".r
+      val alternatePattern = ".*api.soundcloud.com/tracks/(\\d+).*".r
```

Comment:
If we are indeed expecting both - can we just decode and then have a single pattern?